var/spack/repos/builtin/packages/lvm2/package.py

```diff
@@ -35,13 +37,11 @@ def url_for_version(self, version):
     depends_on('libaio')
     depends_on('pkgconfig', type='build', when='+pkgconfig')
 
+    conflicts('platform=darwin',
+              msg='lvm2 depends on libaio which does not support Darwin')
```

Comment:
Same for this package

Reply:
Replied above.

Reply:
You're right, confirmed that things don't build on macOS without libaio:
```
checking libaio.h usability... no
checking libaio.h presence... no
checking for libaio.h... no
configure: error: bailing out
```